### PR TITLE
Added social media share buttons to posts

### LINKED ISSUE #51: Social media share buttons for insight posts
As originally suggested by Nick Weaver, add social media share buttons to the insights (blog) posts. This will enable viewers to more easily share our posts and increase the overall viewership of our posts.

We may also want to have a spike into all insights functionality: compared to other consulting company blogs, are there any additional features we should add to ours?

### PR DESCRIPTION
**Pull request by Techneaux**
Added facebook, linkedin, and twitter share buttons to the bottom of posts.

Used only CSS and html within our own site. Did not link to external CSS social media sharing bars as often they introduce additional unwanted user tracking.

Fixes #51

### CODE REVIEW

File: assets/css/main.scss [hunk 1]
```diff
@@ -56,6 +56,77 @@
...
+.share-btn:hover, .share-btn:active, .share-btn:visited {
+  background-color: #798d8f;
+  text-decoration: none;
+}
+
+.share-btn-twitter {
+  background-color: #0087ba;
+}
+
+.share-btn-twitter:hover, .share-btn-twitter:active, .share-btn-twitter:visited {
```
Comment (johnomalley) [inline assets/css/main.scss:110]: this is a lot cleaner if you nest the styles - this is scss

Comment (Techneaux) [reply assets/css/main.scss]: Thanks @johnomalley! How does the latest version look?

Comment (Techneaux) [reply assets/css/main.scss]: And I just nested and cleaned it up even further.

Comment (johnomalley) [reply assets/css/main.scss]: looks great thanks Jay

**Pull request merged**